core/domain/exp_services_test.py

```diff
@@ -1673,6 +1682,17 @@ class ExplorationCommitLogUnitTests(ExplorationServicesUnitTests):
         'post_commit_status': 'public'
     }
 
+    COMMIT_UPDATE_FIRST_PUBLISHED_MSEC_EXP_2 = {
```

Comment:
Can't the 'first_published' change go in the same commit as the one that actually does the publishing? It doesn't feel like a separate thing altogether, it's more like a consequence of publication.